### ISSUE TITLE
[BUG] Toggle not allowing to scroll

### ISSUE THREAD

**Issue by SantiagoCalligari** [palcarazm/bootstrap5-toggle#153]
You can see this bug when you have a list of block toggle buttons in mobile, if you try to scroll you instead will turn on or off the button instead of scrolling

**Comment by palcarazm** [comment]
Hi, 

In mobile devices we listen for touch events. 

When start scrolling inside a toggle you do a touch. 

Normally in the design the toggle does not take up all the screen space.
I can't think of any use cases where that is necessary. You could show a reduced case of your screen. We could probably readapt the design to make it more user friendly.

**Comment by SantiagoCalligari** [comment]
yeah! I figured that out, the implementation of bootstrap 4 doesn't work that way, that's why I thought that it was a bug :/

here is my example

![Screenshot_2023-10-19-15-34-09-495_com.android.chrome.jpg](https://github.com/palcarazm/bootstrap5-toggle/assets/89709491/34e726d8-146b-463c-9288-2044722ab992)

**Comment by palcarazm** [comment]
Okay. It's seem there is enough space to scroll.

**Comment by SantiagoCalligari** [comment]
I've been fixing it, it's a lame that the same thing for bootstrap 4 works as intended, anyways, I love it, thanks for this man
